### PR TITLE
Feature: remind based on work hours

### PR DESCRIPTION
**Pull request by danielma**
This update changes the reviewbot to ask for reviews based on the number of hours since the last touch, rather than checking at specific times of day.

The config is updated to include a `timezone` key for each reviewer, and the bot will only ever notify users who are within the `8am-5pm` work hour range

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Pull Request Review Reminder Bot
 
-Our team wanted to be reminded via Slack at regular intervals during the day (2 specific times each day)
-about pending Pull Requests, so this little Ruby rake task was created.
+Our team wants to be reminded via Slack to about pending pull requests
```
Comment (zhubert) [inline README.md]: extraneous `to` in `via Slack to about`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+after a certain number of man hours without activity. We calculate man
```
Comment (shanebonham) [inline README.md]: > man hours

Are woman hours factored in too?

**Review by zhubert — APPROVED** [review]
Love it.

File: README.md (hunk 2 above)
Comment (zhubert) [reply README.md]: I was thinking the same thing actually, "work hours" should be clear enough.

Comment (danielma) [reply README.md]: My bad. I meant man as in species, but it's a good point to change it

**Comment by seven1m** [comment]
❤️

File: README.md (hunk 2 above)
Comment (shanebonham) [reply README.md]: OH, YOU MEAN _HUWOMAN BEINGS_? :trollface:

**Pull request merged**